include/nbl/builtin/hlsl/radix_fft.hlsl

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_BUILTIN_HLSL_RADIX_FFT_INCLUDED_
+#define _NBL_BUILTIN_HLSL_RADIX_FFT_INCLUDED_
+
+#include <nbl/builtin/hlsl/complex.hlsl>
+
+
+namespace nbl
+{
+namespace hlsl
+{
+
+const float64_t PI = 3.1415926535897932384626433832795028841971693993751058209749445923078164062862089986280348253421170679;
```

Comment:
please make a `numbers.hlsl` a counterpart to https://en.cppreference.com/w/cpp/numeric/constants